### PR TITLE
feat(#minor); gains-trade; upgrade to perpfutures schema 1.3.1

### PR DESCRIPTION
**Pull request by dhruv-chauhan**
Deployments:
- polygon: https://okgraph.xyz/?q=QmSNxoxL5PFocyRRja2W2mzfQLD8FNx6nC5P9Hcpr4Pe9e
- arbitrum: https://okgraph.xyz/?q=QmdpnsppGcKghriqXt6A92grnaGG4T93FS14rMURTKdXyb

### CODE REVIEW

**Comment by ishraq8** [comment]
syncing data matches the daily financial data. may need to update the visualizer to include `stakeSideRevenue` on pool level but the data is accurate.

File: subgraphs/gains-trade/src/sdk/protocols/perpfutures/pool.ts
Comment (dmelotik) [inline subgraphs/gains-trade/src/sdk/protocols/perpfutures/pool.ts:1]: The SDK files should have updates for the versions like you have in the other PR

Comment (dmelotik) [reply subgraphs/gains-trade/src/sdk/protocols/perpfutures/pool.ts:1]: Otherwise everything looks good

**Review by dmelotik — APPROVED** [review]
lgtm

**Comment by ishraq8** [comment]
total revenue=protocol+supply+stake

a lot of these perps explicitly divide revenue into protocol treasury and staked governance tokens, we wanted to capture that in the subgraph as it is useful to determine where is the money flowing.

**Comment by ishraq8** [comment]
@dmelotik whenever you can, could you add to hosted?

**Pull request merged**